hw/formal/tools/jaspergold/fpv.tcl

```diff
@@ -25,6 +25,9 @@ if {$env(COV) == 1} {
   -exclude_bind_hierarchies
 }
 
+if {$env(DUT_TOP) == "rv_plic_tb"} {
```

Comment:
I really don't like putting it in this file (already full of other junk, but we shouldn't make it worse! :-D)

Can't you stick this in the rv_plic `coverage_waivers.tcl` file that exists already?

Reply:
Not because we first elaborate and analyze the design and then load the after_load files. Having that in waivers file will initialize the coverage again with an exclusion of alert_sender which will generate an error

Reply:
Sorry, there is a way to waive an instance after initialization
check_cov -waiver -add -instance {instance} -comment {comment}

Problem is that the -instance doesn't allow regexp. Based on NumAlerts parameter, we can have more instances of prim_alert_sender. We can do that through tcl loop iterating through NumAlerts and generating check_cov -waiver command per iteration.